connector-runtime/spring-boot-starter-camunda-connectors/src/main/java/io/camunda/connector/runtime/ConnectorsAutoConfiguration.java

```diff
@@ -0,0 +1,155 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH
+ * under one or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information regarding copyright
+ * ownership. Camunda licenses this file to you under the Apache License,
+ * Version 2.0; you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.camunda.connector.runtime;
+
+import io.camunda.client.impl.CamundaObjectMapper;
+import io.camunda.client.spring.properties.CamundaClientProperties;
+import io.camunda.connector.api.secret.SecretProvider;
+import io.camunda.connector.document.jackson.JacksonModuleDocumentSerializer;
+import io.camunda.connector.feel.FeelEngineWrapper;
+import io.camunda.connector.feel.jackson.JacksonModuleFeelFunction;
+import io.camunda.connector.jackson.ConnectorsObjectMapperSupplier;
+import io.camunda.connector.runtime.core.secret.SecretProviderAggregator;
+import io.camunda.connector.runtime.core.secret.SecretProviderDiscovery;
+import io.camunda.connector.runtime.outbound.OutboundConnectorRuntimeConfiguration;
+import io.camunda.connector.runtime.secret.ConsoleSecretProvider;
+import io.camunda.connector.runtime.secret.EnvironmentSecretProvider;
+import io.camunda.connector.runtime.secret.console.ConsoleSecretApiClient;
+import io.camunda.connector.runtime.secret.console.JwtCredential;
+import java.net.URL;
+import java.time.Duration;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Optional;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.autoconfigure.AutoConfiguration;
+import org.springframework.boot.autoconfigure.AutoConfigureBefore;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnMissingBean;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Import;
+import org.springframework.core.env.Environment;
+
+@AutoConfiguration
+@AutoConfigureBefore({
+  OutboundConnectorsAutoConfiguration.class,
+  InboundConnectorsAutoConfiguration.class
+})
+@EnableConfigurationProperties(ConnectorProperties.class)
+@Import(OutboundConnectorRuntimeConfiguration.class)
```

Comment:
Curious why this is needed?

Shouldnt we for example also add the `ValidationProvider` this config here as well as its used by all Connectors (inbound/ooutbound)?